muesli/web/templates/Fragments/menu.pt

```diff
@@ -1,21 +1,92 @@
+<!--! -*- coding: utf-8 -*-                                                 -->
+<!--! muesli/web/templates/Fragments/menu.pt                   -->
+<!--                                                                        -->
+<!--! This file is part of MUESLI.                                          -->
+<!--! Copyright (C) 2020, Christian Heusel <christian (at) heusel.eu>       -->
+<!--                                                                        -->
+<!--! This program is free software: you can redistribute it and/or modify  -->
+<!--! it under the terms of the GNU General Public License as published by  -->
+<!--! the Free Software Foundation, either version 3 of the License, or     -->
+<!--! (at your option) any later version.                                   -->
+<!--                                                                        -->
+<!--! This program is distributed in the hope that it will be useful,       -->
+<!--! but WITHOUT ANY WARRANTY; without even the implied warranty of        -->
+<!--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         -->
+<!--! GNU General Public License for more details.                          -->
+<!--                                                                        -->
+<!--! You should have received a copy of the GNU General Public License     -->
+<!--! along with this program.  If not, see <http://www.gnu.org/licenses/>. -->
+
 <metal:main metal:define-macro="menu">
-  <ul class="menu" id="menu-left">
-    <div tal:omit-tag="" tal:condition="request.userInfo.is_loggedin()">
-      <li><a tal:attributes="href request.route_path('start')">Übersicht</a></li>
-      <li tal:condition="request.userInfo.is_admin()"><a tal:attributes="href request.route_path('admin')">Administration</a></li>
+  <nav class="navbar navbar-dark navbar-expand-md bg-dark justify-content-md-center justify-content-start">
+      <div class="dropdown">
+        <a href="#" class="display-1 navbar-brand" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          <img class="mr-1" style="height: 1.5em;" tal:attributes="src request.route_path('muesli.web:static/favicon.ico')">
+          MÜSLI
+        </a>
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+          <a class="dropdown-item" tal:attributes="href request.route_path('start')">
+            <img class="mr-1" style="height: 1em;" tal:attributes="src request.route_path('muesli.web:static/favicon.ico')">
```

Comment:
Seconded. I am always a little baffled when I click on the icon and don't get to the front page.
If the dropdown should be kept, maybe add a little downwards arrow next to the MÜSLI icon and the following logic:
* Clicking the icon leads to overview
* Clicking the arrow opens the dropdown

Reply:
I would prefer the version which @TuringTux suggested... I got the idea from an other program I recently had to use (Jira/Confluence :vomiting_face:) and liked it as a feature ... The MaMpf developer also liked the idea of linking the two envs a little closer, especially since they will look more similar in the future.